File: problems/3005/Solution.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] freq = new int[101];
        int best = 0;
        for (int x : nums) best = Math.max(best, ++freq[x]);
        int ans = 0;
        for (int count : freq) if (count == best) ans += count;
        return ans;
    }
}

File: problems/3005/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxFrequencyElements = function(nums) {
    const freq = new Map();
    let best = 0;
    for (const x of nums) {
        const count = (freq.get(x) || 0) + 1;
        freq.set(x, count);
        best = Math.max(best, count);
    }
    let ans = 0;
    for (const count of freq.values()) if (count === best) ans += count;
    return ans;
};
